apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/admin/api/client/BftScanConnection.scala

```diff
@@ -139,10 +139,17 @@ class BftScanConnection(
               "refresh_scan_list",
               "refresh_scan_list",
               bft.refresh(this)(tc).flatMap { connections =>
-                if (connections.failed > 0)
+                val failed = connections.failed
+                val total = connections.totalNumber
+                val f = connections.f
+                if (connections.failed > connections.f)
```

Comment:
Hm should this be f or 2f+1? I thought we want 2f+1 working connections so we can read from them and tolerate f failures.